### PR TITLE
Kaltura Source doc 

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206238447333773

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kaltura-source.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/files/c2c/kaltura/example.json';
+import MyComponentSource from '!!raw-loader!/files/c2c/kaltura/example.json';
+import TerraformExample from '!!raw-loader!/files/c2c/kaltura/example.tf';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/kaltura-logo.png')} alt="icon" width="70"/>
+
+Kaltura is a video platform for modular systems that exposes different web-services and that may be deployed in several deployment modes to support different levels of scale. Kaltura’s platform comes in different editions, including the Kaltura-hosted SaaS edition, managed by Kaltura for single publishers and Value Added Resellers (VARs), as well as in several licensing modes of the, self-hosted, Kaltura On-Prem edition: Kaltura Community Edition, Kaltura On-Prem for Publishers and Kaltura OnPrem for OEMs. Kaltura source collects audit trail events and base entry events data and sends it to Sumo Logic.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kaltura-source.md:19]:
```suggestion
Kaltura is a video platform for modular systems that exposes different web services that may be deployed in several deployment modes to support different levels of scale. Kaltura’s platform comes in different editions, including the Kaltura-hosted SaaS edition, managed by Kaltura for single publishers and Value Added Resellers (VARs). Kaltura also offers several licensing modes of the self-hosted Kaltura On-Prem edition: Kaltura Community Edition, Kaltura On-Prem for Publishers, and Kaltura OnPrem for OEMs. Kaltura source collects audit trail events and base entry events data and sends it to Sumo Logic.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kaltura-source.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+| Polling Interval | Data |
+| :--- | :--- |
+| 5 min |  [Audit Trail Events](https://developer.kaltura.com/api-docs/service/auditTrail/action/list) |
+| 1 hour |  [Base Entry Events](https://developer.kaltura.com/api-docs/service/baseEntry/action/list) |
+
+## Setup
+
+### Vendor configuration
+
+In this configuration, you will create a new [Kaltura App Token](https://developer.kaltura.com/api-docs/VPaaS-API-Getting-Started/application-tokens.html) in the [Kaltura Develops Portal](https://developer.kaltura.com/api-docs/service/appToken/action/add) or via your own hosted API portal to generate a App token and App Id.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kaltura-source.md:32]:
```suggestion
In this configuration, you will create a new [Kaltura App Token](https://developer.kaltura.com/api-docs/VPaaS-API-Getting-Started/application-tokens.html) in the [Kaltura Developer Portal](https://developer.kaltura.com/api-docs/service/appToken/action/add) or via your own hosted API portal to generate an app token and app ID. 
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kaltura-source.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+### Vendor configuration
+
+In this configuration, you will create a new [Kaltura App Token](https://developer.kaltura.com/api-docs/VPaaS-API-Getting-Started/application-tokens.html) in the [Kaltura Develops Portal](https://developer.kaltura.com/api-docs/service/appToken/action/add) or via your own hosted API portal to generate a App token and App Id. 
+
+#### Create a New App Token
+
+A Kaltura App Token with specific permissions is required for Sumo Logic to access Audit Trail and Base Entry Events from Kaltura. Follow the below instructions to create a new App Token.
+
+1. Start a [Kalutra session](https://developer.kaltura.com/api-docs/service/session/action/start) using an admin account.
+1. Enter the following parameter in the body:
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kaltura-source.md:39]:
```suggestion
1. Enter the following parameters in the body:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kaltura-source.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    - **hashType**. SHA1[SHA1] - Kaltura App Token Hash Type.
+    - **sessionType**. ADMIN[2] - Type of Kaltura Session that will be created using the current token.
+1. Click on the **Send Request**.
+1. Copy and save the **App Token**, **App Token ID**, and **Partner Id** from the response.
+
+### Source configuration
+
+When you create an Kaltura source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+To configure an Kaltura source:
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kaltura-source.md:63]:
```suggestion
To configure a Kaltura source:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kaltura-source.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+1. **Partner ID**. Enter the **Partner ID** collected from the [Vendor configuration](#create-a-new-app-token).
+1. **App Token ID**. Enter the **App Token ID** collected from the [Vendor configuration](#create-a-new-app-token).
+1. **App Token**. Enter the **App Token** collected from the [Vendor configuration](#create-a-new-app-token).
+1. **Select Base Entry Types for Base Entry Logs**. You have the option to **Collect all types** or **Select types**, where you can specify the exact event categories you would like to collect from the base entry logs. Must select from the pre-defined list.
+1. **Polling Interval** You have the option to select how often to poll for base entry events. Default is 24 hours.
+1. When you are finished configuring the Source, click **Save**.
+
+## JSON schema
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kaltura-source.md:82]:
```suggestion
Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [Use JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for details. 
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kaltura-source.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+| config | JSON Object | [Configuration object](#configuration-object) | Yes | Source type specific values. |
+
+### Configuration Object
+
+| Parameter | Type | Required | Default | Description | Example |
+|:--|:--|:--|:--|:--|:--|
+| name | String | Yes | `null` | Type a desired name of the source. The name must be unique per Collector. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_source`. | `"mySource"` |
+| description | String | No | `null` | Type a description of the source. | `"Testing source"`
+| category | String | No | `null` | Type a category of the source. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_sourceCategory`. See [best practices](/docs/send-data/best-practices) for details. | `"mySource/test"`
+| fields | JSON Object | No | `null` | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field _siemForward to enable forwarding to SIEM.|`{"_siemForward": false, "fieldA": "valueA"}` |
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kaltura-source.md:97]:
```suggestion
| fields | JSON Object | No | `null` | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field `_siemForward` to enable forwarding to SIEM.|`{"_siemForward": false, "fieldA": "valueA"}` |
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812. Approved contingent on your addressing my suggested updates.

**Comment by JV0812** [comment]
Thanks for your review @jpipkin1. I have incorporated your comments.

**Comment by jpipkin1** [comment]
Merging at the request of @yleiferman.

**Pull request merged**